Dell/recovery_backend.py

```diff
@@ -825,6 +825,12 @@ def _prepare_reboot(self, dest):
         if ret is not 0:
             raise RestoreFailed("error setting one time grub entry")
 
+        if reboot:
+            logging.debug("Prepare to reboot using gdbus")
+            ret = subprocess.call(['/usr/bin/gdbus', 'call', '-y', '-d', 'org.freedesktop.systemd1', '-o', '/org/freedesktop/systemd1', '-m', 'org.freedesktop.systemd1.Manager.Reboot'])
+            if ret is not 0:
+                raise RestoreFailed("error invoking gdbus to reboot")
+
```

Comment:
Don't shell out to the gdbus to make a dbus call, use the native dbus functionality or shell out to the reboot command.

Reply:
> Don't shell out to the gdbus to make a dbus call, use the native python dbus functionality or shell out to the reboot command. Either way is fine to me.

Per your suggestion, I've revised the call to use reboot command, instead of gdbus call.  https://github.com/dell/dell-recovery/pull/38/commits/15ce9587bc44dc09298ef3f519bb34bb577bed43